hr_skill/views/hr_skill.xml

```diff
@@ -43,6 +43,7 @@
     <menuitem name="Skills"
               parent="hr.menu_human_resources_configuration"
               id="menu_open_view_skill_form"
-              action="open_view_skill_form"/>
+              action="open_view_skill_form"
+              groups="base.group_no_one"/>
```

Comment:
This menu item will only be visible if the parent Configuration menu item also is.
So IMO this change in unnecessary.